risc0/zkvm/src/host/client/prove/bonsai.rs

```diff
@@ -22,6 +22,9 @@ use super::Prover;
 use crate::{ExecutorEnv, ProverOpts, Receipt, VerifierContext};
 
 /// An implementation of a [Prover] that runs proof workloads via Bonsai.
+///
+/// Requires a BONSAI_API_URL and BONSAI_API_KEY environment variables to
+/// submit proving sessions to Bonsai.
```

Comment:
```suggestion
/// Requires BONSAI_API_URL and BONSAI_API_KEY environment variables to
/// submit proving sessions to Bonsai.
```

Reply:
Also would be good to wrap env var names with backticks for formatting.

Reply:
Good point
```suggestion
/// Requires `BONSAI_API_URL` and `BONSAI_API_KEY` environment variables to
/// submit proving sessions to Bonsai.
```